### PR TITLE
GRpc Curation Service

### PR DESCRIPTION
**Pull request by harryurban**
add protobufs for coremedia curation service

### CODE REVIEW

File: stroeer/coremedia/v1/curation-service.proto [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+package stroeer.coremedia.v1;
+
+import "stroeer/coremedia/v1/curated-item.proto";
+
+option java_multiple_files = true;
+option java_package = "de.stroeer.coremedia.v1";
+option go_package = "github.com/stroeer/tapir/go/stroeer/coremedia/v1;coremedia";
+
+service CurationService {
+  rpc GetCuratedList (GetCuratedListRequest) returns (stream stroeer.coremedia.v1.CuratedItem) {}
```
Comment (harryurban) [inline stroeer/coremedia/v1/curation-service.proto:12]: notice the return type `stream` vs a Wrapper object.
this isn't actually a situation that `stream` is meant for, what do you say? wrapper or stream?

File: stroeer/coremedia/v1/curated_item.proto [hunk 2]
```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package stroeer.coremedia.v1;
```
Comment (moritzzimmer) [inline stroeer/coremedia/v1/curated_item.proto:3]: blank line after `package`.

File: stroeer/coremedia/v1/curated_item.proto [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+option java_multiple_files = true;
```
Comment (moritzzimmer) [inline stroeer/coremedia/v1/curated_item.proto:4]: in the other proto files, the option list is ordered lexicographically

File: stroeer/coremedia/v1/curated_item.proto [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+option java_package = "de.stroeer.coremedia.v1";
+option go_package = "github.com/stroeer/tapir/go/stroeer/coremedia/v1;coremedia";
+
+message CuratedItem {
```
Comment (moritzzimmer) [inline stroeer/coremedia/v1/curated_item.proto:8]: `CuratedArticle`?

File: stroeer/coremedia/v1/curation_service.proto [hunk 5]
```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package stroeer.coremedia.v1;
+
+import "stroeer/coremedia/v1/curated_item.proto";
+
+option java_multiple_files = true;
```
Comment (moritzzimmer) [inline stroeer/coremedia/v1/curation_service.proto:7]: in the other proto files, the option list is ordered lexicographically

**Pull request merged**